aws-py-eks/vpc.py

```diff
@@ -0,0 +1,104 @@
+from pulumi_aws import ec2
+
+REGION = "us-east-2"
+
+## VPC
+
+vpc = ec2.Vpc(
+    'eks-vpc', 
+    cidr_block='10.100.0.0/16',
+    instance_tenancy='default',
+    enable_dns_hostnames=True,
+    enable_dns_support=True,
+    tags={
+        'Name' : 'pulumi-eks-vpc',
+        'kubernetes.io/cluster/pulumi-eks-cluster' : 'shared'
+    }
+)
+
+igw = ec2.InternetGateway(
+    'vpc-ig',
+    vpc_id=vpc.id,
+    tags={
+        'Name' : 'pulumi-vpc-ig'
+    }
+)
+
+## Subnets
+
+vpc_subnet_1 = ec2.Subnet(
```

Comment:
based on aws.get_availability_zones, we should be able to loop and create the subnets for each of the zones, rather than hardcoding